### DISCUSSION TITLE
Is it possible to run Intel macOS guest on an Intel macOS host?

### DISCUSSION THREAD

**Discussion by TomasHubelbauer** [utmapp/UTM#4397] [UTM Help (macOS)]
I searched for Intel related issues and PRs and also checked out the Guides and Gallery on the site, but I am still left wondering: Can UTM be used to run an ephemeral Intel macOS guest VM on an Intel macOS host OS? I found a lot of information related to the Arm macOS versions as well as Windows. But I have not found a clear answer to this question, although I have a feeling the answer may be no? In a nutshell, I am looking for something akin to the Windows Sandbox but for macOS. There is no comparable native feature like that, so I turned my attention towards UTM.

I am aware of using disk partitions to install two macOS installations side-by-side, BTW. I am looking for a solution which would allow me to work in a macOS-in-macOS style instead so I could quickly context switch between the main and the ephemeral/utility installation.

**Configuration**  
* UTM Version: 3.2.4 (58)
* macOS Version: 12.5.1 (21G83)
* Mac Chip (Intel, M1, ...): Intel

**Comment by CodeRunner5235** [comment]
I believe that UTM is only for ARM devices, though I could be wrong. However, is you want to run intel macOS inside intel macOS, you might be able to use something like VMware or Virtualbox. They will let you virtualize other OSes, however, they will only let you run OS that are for x86_x64  systems. Useing a pure QEMU install is also an option. QEMU is what UTM uses to virtualize/emulate other systems. Its main trick is, unlike VMware and Virtualbox, it can virtualize x86_x64, but it can also emulate other system architectures as well (ARM, RISC, PPC, etc...) My only issue with purely using QEMU is that it is much harder to use, as everything is done in the command line.

**Comment by osy** [comment]
People have gotten it working but I recommend against it because QEMU support is incomplete.

**Reply by mariobrostech** [reply]
Sorry to bump an old thread, but has QEMU's support for macOS guests on Intel hosts improved since you wrote this, to your knowledge? Is there any hope of this being added in the future? I'd love to use UTM as an alternative to VMware Fusion, but my workflow requires being able to use macOS guest VMs. Could an experimental option (similar to how GPU acceleration with Linux is currently handled) be added for those who accept the risk and are willing to try it out anyway?

Also, if there are any posts from people who have gotten this working on their own that you'd be able to link to, please do. I'm very interested in learning more about this!

Thanks in advance!

**Reply by 4n0th3rm3** [reply]
I had it running on Proxmox, macOS installation should be possible - didn't boot with UTM for me but actually some month passed and I can't remember the issue...
https://github.com/kholia/OSX-KVM

**Reply by CuriousTommy** [reply]
@osy Do you have a list of requirements that QEMU would need to support before UTM can offer official x86 macOS guest support?

**Comment by tifasoftware** [comment]
You think you might need to use OpenCore (used on hackintosh) to get it working on QEMU.
